src/services/unlockCourtCase.ts

```diff
@@ -13,11 +12,15 @@ const unlockCourtCase = async (
   courtCaseId: number,
   user: User,
   unlockReason: UnlockReason
-): Promise<UpdateResult | Error> => {
+): Promise<UpdateResult | Error | undefined> => {
```

Comment:
This is not pretty, but we are going to refactor the return types of all the service functions in the next PR